docs/develop/java/spring/redis-om/index-redis-om-spring.mdx

```diff
@@ -0,0 +1,53 @@
+---
+id: redis-om-spring
+title: Redis OM - Spring
+sidebar_label: Introduction
+slug: /develop/java/spring/redis-om/redis-om-spring
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import RedisCard from '@site/src/theme/RedisCard';
+
+
+## Introduction
+
+The Redis OM family of projects aim is to provide high-level abstractions idiomatically implemented for
+ your language/platform of choice. We currently cater to the Node, Python, .Net and Spring communities.
```

Comment:
stray space

Reply:
Also, I think .NET is capitalized.

Reply:
And Node.js is always written Node.js but spoken as Node. From their style guide.